jenkins/opensearch/benchmark-test.jenkinsfile

```diff
@@ -70,6 +106,11 @@ pipeline {
             description: 'Use OpenSearch min distribution',
             defaultValue: false
         )
+        booleanParam(
+            name: 'ENABLE_REMOTE_STORE',
```

Comment:
This should be setup in cron as well:
`ENABLE_REMOTE_STORE`

Reply:
Added.